pkg/registry/node.go

```diff
@@ -49,6 +51,11 @@ func (node *Node) BuildClient(
 	dialOpts := append([]grpc.DialOption{
 		grpc.WithTransportCredentials(creds),
 		grpc.WithDefaultCallOptions(),
+		grpc.WithKeepaliveParams(keepalive.ClientParameters{
```

Comment:
In `server.go` we have a `KeepAliveEnforcementPolicy` set with a `MinTime` of 15 seconds. I believe we will start booting clients who do keepalives more frequently than that. 30 secs should be fine

Reply:
Nice catch